Chat/WebApi/Startup.cs

```diff
@@ -29,5 +30,32 @@ public static void ConfigureValidation(this IServiceCollection services)
             services.AddFluentValidationAutoValidation();
             services.AddValidatorsFromAssembly(Assembly.GetExecutingAssembly());
         }
+
+        public static string GetConfiguredConnectionString(this IConfiguration configuration, string variant="Default")
+        {
```

Comment:
you need to create separate appsettings files. One for production (for docker) and another one for development and store in these files all necessary configurations. This way you you dont need the GetConfiguredConnectionString() as it will be automatically selected the necessary appsettings file with with the correct connection strings. You just need to set the environment in docker compose file for example:   
environment:
      - ASPNETCORE_ENVIRONMENT=Docker